sdks/wasm/sdk/WebAssembly.Net.WebSockets/WebAssembly.Net.WebSockets.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>Mono.WebAssembly ClientWebSocket implementation. This package is not intended to be referenced directly.</Description>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <!-- get rid of warnings about assemblies not in lib -->
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="..\..\..\wasm\framework\WebAssembly.Net.WebSockets.dll" PackagePath="lib\netstandard2.0\%(Filename)%(Extension)" Pack="True" />

```

Comment:
Maybe just use a wildcard here, WebAssembly.Net.WebSockets.*

Reply:
I will be changing this soon.  Thanks for the heads up.

Reply:
Should be modified now